.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: java
 jdk:
-  - oraclejdk8
+- oraclejdk8
 addons:
   sonarcloud:
-    organization: "improbable-keanu"
-script:
-  - ./gradlew check
-  - 'if [[ ! -z $SONAR_TOKEN ]]; then sonar-scanner; fi'
+    organization: improbable-keanu
```

Comment:
I don't think the quote marks make a difference, do they?

Reply:
Nah - travis auto-deleted them when I added the encrypted file